dotnet/samples/Concepts/Search/GoogleTextSearchExample.cs

```diff
@@ -0,0 +1,170 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json;
+using Google.Apis.Http;
+using Microsoft.SemanticKernel.Plugins.Web.Google;
+using Microsoft.SemanticKernel.Search;
+
+namespace Search;
+
+/// <summary>
+/// This example shows how to create and use a <see cref="GoogleTextSearch"/>.
+/// </summary>
+public class GoogleTextSearchExample(ITestOutputHelper output) : BaseTest(output)
+{
+    /// <summary>
+    /// Show how to create a <see cref="GoogleTextSearch"/> and use it to perform a text search.
+    /// </summary>
+    [Fact]
+    public async Task UsingGoogleTextSearchAsync()
+    {
+        // Create an ITextSearch instance using Google search
+        var textSearch = new GoogleTextSearch(
+            initializer: new() { ApiKey = TestConfiguration.Google.ApiKey, HttpClientFactory = new CustomHttpClientFactory(this.Output) },
+            searchEngineId: TestConfiguration.Google.SearchEngineId);
+
+        var query = "What is the Semantic Kernel?";
+
+        // Search and return results as string items
+        KernelSearchResults<string> stringResults = await textSearch.SearchAsync(query, new() { Count = 4, Offset = 0 });
+        Console.WriteLine("--- String Results ---\n");
+        await foreach (string result in stringResults.Results)
+        {
+            Console.WriteLine(result);
+            Console.WriteLine("------------------------------------------------------------------------------------------------------------------");
```

Comment:
Shorter breaks, IMHO.

```suggestion
            Console.WriteLine("------------------");
```

Reply:
In test cases, seemed generally we could use `ITestoutputHelper.Output.WriteLine()` for output,
and maybe we could use `new string('-', 50)` to avoid too long code line

https://xunit.net/docs/capturing-output

Reply:
@WeihanLi our Concepts project `Console` is not the actual `System.Console` is a pointer to the `ITextoutputHelper`. We kept the `Console` to make it easier for users to copy and paste that code into `COnsole` applications.

Reply:
@markwallace-microsoft This might look nice in a very large console window, but not nice in a small one (wrapping)